Doppler.HelloMicroservice/Startup.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+
+namespace Doppler.HelloMicroservice
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+
+            services.AddControllers();
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "Doppler.HelloMicroservice", Version = "v1" });
```

Comment:
@ms-darianbenito remind me that it will not work if we expose the API inside a segment (not directly in the domain), example: `https://apis.fromdoppler.com/hello/v1/swagger.json` vs `https://hello.fromdoppler.com/v1/swagger.json`

They already fixed it in _doppler-email-address-validator_, I have to apply the same change here.

https://github.com/MakingSense/doppler-email-address-validator/blob/2f16285d595b51879635d6aaa6f7968fcb109065/Doppler.EmailAddressValidator.Api/Startup.cs#L47-L51

Reply:
Done!

And tested:

![image](https://user-images.githubusercontent.com/1157864/114596097-92980b80-9c65-11eb-8d3f-c812baaf567d.png)